pkg/machines/libvirt-virsh.js

```diff
@@ -291,7 +291,7 @@ LIBVIRT_PROVIDER = {
         logDebug(`${this.name}.GET_STORAGE_VOLUMES(connectionName='${connectionName}', poolName='${poolName}')`);
         const connection = VMS_CONFIG.Virsh.connections[connectionName].params.join(' ');
         // Caution: output of virsh v1.3.1 (ubuntu-1604) and v3.7.0 differs, the 'grep' unifies it
-        const command = `virsh ${connection} -q pool-refresh ${poolName} && virsh ${connection} -q -r vol-list ${poolName} --details | (grep file || true)`;
+        const command = `virsh ${connection} -q pool-refresh ${poolName} && virsh ${connection} -q -r vol-list ${poolName} --details`;
```

Comment:
This is a welcome cleanup, as it's not clear what this does in the first place. But can you please also adjust the comment, as it doesn't apply any more?

Reply:
Apart from filtering out all that is not of type file (which is not already correct), the grep there unifies output between older a newer versions of virsh.
But no distro version cockpit currently supports runs on libvirt/virsh version so old, that it has different output. So there is no need for the "unification".
So I decided to remove whole comment, since it's not relevant anymore.